mozetl/maudau/maudau.py

```diff
@@ -0,0 +1,214 @@
+"""
+Calculate Firefox monthly- and daily-active users.
```

Comment:
Please add a link to the original notebook or bug number for documentation purposes.